docs/docs-beta/docs/tutorials/category-two/llm-fine-tuning/index.md

```diff
@@ -0,0 +1,133 @@
+---
+title: LLM Fine Tuning
+description: Learn how to fine tune an LLM
+last_update:
+   author: Dennis Hume
+sidebar_position: 10
+sidebar_class_name: hidden
+---
+
+# Fine-Tune an LLM
+
+In this tutorial, you'll build an pipeline with Dagster that:
+
+- Imports data from Goodreads DuckDB
+- Does feature engineering to enhance the data
+- Creates and validates a training and validation file
+- Executes an OpenAI fine-tuning job and validates the model
+
+## You will learn to:
+
+- Set up a Dagster project with the recommended project structure
+- Create and materialize assets
+- Create and materialize dependant assets
+- Ensure data quality with asset checks
+
+<details>
+  <summary>Prerequisites</summary>
+
+To follow the steps in this guide, you'll need:
+
+- Basic Python knowledge
+- Python 3.9+ installed on your system. Refer to the [Installation guide](/getting-started/installation) for information.
+- Familiarity with SQL and Python data manipulation libraries, such as Pandas.
+- Understanding of data pipelines and the extract, transform, and load process.
+</details>
+
+
+## Step 1: Set up your Dagster environment
+
+First, set up a new Dagster project.
+
+1. Open your terminal and create a new directory for your project:
+
+   ```bash
+   mkdir dagster-llm-fine-tune
+   cd dagster-llm-fine-tune
+   ```
+
+2. Create and activate a virtual environment:
+
+   <Tabs>
+   <TabItem value="macos" label="MacOS">
+   ```bash
+   python -m venv dagster_tutorial
+   source dagster_tutorial/bin/activate
```

Comment:
As an aside, I wonder if we can take the docs-revamp as an opportunity to use `uv venv` here.